### ISSUE TITLE
[Suggestion] Plugin-Hider plugins bypass

### ISSUE THREAD

**Issue by NilanFuncke** [MeteorDevelopment/meteor-client#4449]
### Describe the feature

Please add bypass for servers with plugin hider plugins

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by Wide-Cat** [comment]
Tell us what the bypass is and we'll add it

**Comment by Lapinoub** [comment]
I don't think that it is possible to bypass as the plugin-hider basically blocks the server from sending the plugins. The only way you can know the plugins is by analyzing non vanilla behaviour
